pkg/network/tracer/connection/tracer.go

```diff
@@ -304,6 +308,7 @@ func (t *tracer) GetConnections(buffer *network.ConnectionBuffer, filter func(*n
 	// Iterate through all key-value pairs in map
 	key, stats := &netebpf.ConnTuple{}, &netebpf.ConnStats{}
 	seen := make(map[netebpf.ConnTuple]struct{})
+	conns := make(map[netebpf.ConnTuple]*network.ConnectionStats)
```

Comment:
Any concerns about memory increase because of this?

Reply:
Very small increase on the load testing cluster, but I will do some more testing.
![system-probe working set (avg) (2)](https://github.com/DataDog/datadog-agent/assets/6599778/691b4cd1-661b-47cc-803f-3c155aeef058)